NAMESPACE

```diff
@@ -2,5 +2,7 @@
 
 export("%>%")
 export(read_compare_csv)
+export(read_gather)
 export(read_taxonomy_annotate)
+export(taxonomy_annotate_to_metacoder)
 importFrom(magrittr,"%>%")
```

Comment:
I assume these functions are replicating what is demonstrated in your previous blog posts : ?
https://taylorreiter.github.io/2022-07-28-From-raw-metagenome-reads-to-phyloseq-taxonomy-table-using-sourmash-gather-and-sourmash-taxonomy/ 
https://taylorreiter.github.io/2022-08-30-Visualizing-the-taxonomic-composition-of-metagenomes-using-sourmash-and-metacoder/ 

Just to help with my comparisons?

Reply:
They replicate a piece of it, yes. Essentially they get the sourmash taxonomy output into a metacoder object. This notebook is maybe a better representation of what they do: https://github.com/Arcadia-Science/2022-prjna853785-sourmash/blob/main/notebooks/20220901-sourmash-taxonomy-as-metacoder-heat-tree.ipynb

Already with tests and stuff i've found errors in the prjna853785 notebooks and in my blog posts 🤦‍♀️ so i'm really happy about getting all of these scripts into a package with testing, I think they'll streamline the viz process and guard against errors now and later